stores/json/store.go

```diff
@@ -42,15 +42,25 @@ func (store BinaryStore) EmitEncryptedFile(in sops.Tree) ([]byte, error) {
 	return store.store.EmitEncryptedFile(in)
 }
 
+const NOT_BINARY_HINT = ("This is likely not an encrypted binary file?" +
+	" If not, use --output-type to select the correct output type.")
```

Comment:
As the store can be used while using SDK, mentioning of a flag can be misleading.

Reply:
I guess then it's better to use a specialized error, which allows to print the message when returned in the CLI.

I'll update the PR accordingly (though likely not today :) ).